src/java/main/org/apache/zookeeper/server/command/MonitorCommand.java

```diff
@@ -75,9 +79,9 @@ public void commandRun() {
             print("synced_followers", leader.getForwardingFollowers().size());
             print("pending_syncs", leader.getNumPendingSyncs());
 
-            print("last_proposal_size", leader.getProposalStats().getLastProposalSize());
-            print("max_proposal_size", leader.getProposalStats().getMaxProposalSize());
-            print("min_proposal_size", leader.getProposalStats().getMinProposalSize());
+            print("last_proposal_size", leader.getProposalStats().getLast());
```

Comment:
Infact, getLastProposalSize, getMaxProposalSize, getMinProposalSize is more meaningful. Can we keep this names?

Reply:
Sure, no problem. I thought it was redundant to refer the class name in method names, but I'll revert that.

Reply:
One more observation. It seems "ProposalStats" is reused for clientResponseStats metrics, but it may create confusions due to the method name mismatches. Again, ProposalStats javadocs says "Provides live statistics about a running Leader."  Also, stats will evolve and could be chance of adding unrelated metrics later. 

How about create a new class "ResponseStats" and name metrics like below. If we look at the existing min_proposal_size metrics, they didn't use the term leader/quorum. Keeping that in mind, do we need specifically ''client'' term in the metrics, simply response gives a context to the users that server-to-client response. Whats your opinion?
```
last_response_size,
max_response_size,
min_response_size
```

Reply:
Agreed, javadoc is misleading.

Actually I wanted to keep these two things together and would like to refactor both in a later commit. They refer to the same thing basically, exposing statistics on Jute buffer usage which is a feed of int values. I'm happy to find a better name for the class, but wouldn't create a separate one for almost the same purpose.

What do you think of `BufferStats`?

Reply:
You're right about stats will evolve with time, but proposal and client request size are common in Jute buffer usage. I have a strong feeling that these two things will always evolve together hence it makes sense to keep the stats together which also makes refactoring easier. Later, if it turns out that they diverge, we can easily split them.